### PR TITLE
Fixes #6 by calculating max table width

### LINKED ISSUE #6: Table width exceeds wanted width...
Read your blog and thought to give it a try.  Love the concept! 

Ran into this when output was generated:
```
../gems/terminal-table-1.8.0/lib/terminal-table/table.rb:305:in `recalc_column_widths': Table width exceeds wanted width of 200 characters. (RuntimeError)
```

Thoughts on toggling the settings within the `skunk` command?  `skunk --width=400`?

https://github.com/fastruby/skunk/blob/98ba2b0a59f67a67004ec413673304ece1c17217/lib/skunk/cli/commands/status_reporter.rb#L64

### PR DESCRIPTION
**Pull request by etagwerker**
Hey @bronzdoc @joshRpowell @danluchi, this PR fixes #6. 

The problem with the table width issue is related to pathnames in someone's application being "too long" for the hardcoded value of 200. Some files are deeply nested and we are displaying all of them in the table.

This solution finds the longest pathname and it uses that length to calculate the total width of
the table.

One side effect: You might end up with quite a wide table if you have files that are deeply nested
and/or have really really long names.

What do you think about this solution?

### CODE REVIEW

File: lib/skunk/cli/commands/status_reporter.rb [hunk 1]
```diff
@@ -66,9 +66,11 @@ def stink_score_average
       end
 
       def table_options
+        max = sorted_modules.max_by { |a_mod| a_mod.pathname.to_s.length }
+        width = max.pathname.to_s.length + 77
```
Comment (bronzdoc) [inline lib/skunk/cli/commands/status_reporter.rb]: What is the `77` magic number?

Comment (etagwerker) [reply lib/skunk/cli/commands/status_reporter.rb]: @bronzdoc This PR makes the width variable. Width is determined by "longest file path" + 74 characters.

```
$ irb
2.6.3 :001 > "| stink_score  | churn_times_cost | churn      | cost       | coverage   |".size
 => 74
```

I guess I could change it so `77` is relative to the right side of the header?

Comment (bronzdoc) [reply lib/skunk/cli/commands/status_reporter.rb]: Maybe we could put that 77 in a constant? `EXTRA_HEADER_LENGTH` ?
```ruby
width = max.pathname.to_s.length + EXTRA_HEADER_LENGTH
```

Comment (etagwerker) [reply lib/skunk/cli/commands/status_reporter.rb]: @bronzdoc Good idea. I'll do that later today.

**Pull request merged**